streams/src/test/java/org/apache/kafka/streams/integration/RegexSourceIntegrationTest.java

```diff
@@ -178,6 +178,44 @@ public void subscribe(final Pattern topics, final ConsumerRebalanceListener list
 
         TestUtils.waitForCondition(() -> assignedTopics.equals(expectedSecondAssignment), STREAM_TASKS_NOT_UPDATED);
 
+        CLUSTER.deleteTopicsAndWait("TEST-TOPIC-1", "TEST-TOPIC-2");
+    }
+
+    @Test
+    public void testRegexRecordsAreProcessedAfterReassignment() throws Exception {
+        final String topic1 = "TEST-TOPIC-1";
+        final String topic2 = "TEST-TOPIC-2";
```

Comment:
nit: we could put topic2 initialization before L196

Reply:
agree